setup.py

```diff
@@ -114,6 +115,9 @@ def build_extension(self, ext):
             ["cmake", "--build", "."] + build_args, cwd=self.build_temp
         )
 
+        # copy nmodl module with shared library to extension directory
+        copy_tree(os.path.join(self.build_temp, 'nmodl'), extdir)
```

Comment:
@ohm314 : what happens is that using

```
 "-DCMAKE_LIBRARY_OUTPUT_DIRECTORY_{}={}".format(cfg.upper(), extdir)
```

We only get nmodl module with python files but not the actual shared library `_nmodl....so`. Looking at Fernando's setup.py, I copied above where he copies the built module directory to extension directory as here: https://github.com/neuronsimulator/nrn/blob/7dc4a6b21b4338e3bfb1d7d09a7270c508a9a195/setup.py#L90